tests/contrib/mue/test_variablelengthhmm.py

```diff
@@ -0,0 +1,70 @@
+import torch
+
+from pyro.contrib.mue.variablelengthhmm import VariableLengthDiscreteHMM
+
+
+def test_hmm_log_prob():
+    torch.set_default_tensor_type('torch.DoubleTensor')
```

Comment:
nit: This should happen automatically due to [conftest.py](https://github.com/pyro-ppl/pyro/blob/dev/tests/conftest.py#L13). We try to avoid setting global variables in individual tests because those can leak across tests.